hooks/use-realtime-transcription.ts

```
@@ -0,0 +1,386 @@
+      case 'input_audio_buffer.speech_started':
+        console.log('Speech started')
+        setCurrentTranscription('Listening...')
+        break
+
+      case 'input_audio_buffer.speech_stopped':
+        console.log('Speech stopped')
+        setCurrentTranscription('Processing...')
+        break
+
+      case 'conversation.item.input_audio_transcription.completed':
+        const transcript = message.transcript
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)